### PR TITLE
New Default Error page #8375

### PR DESCRIPTION
**Pull request by rymsha**

### CODE REVIEW

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageRichBuilder.java  5
- modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageSimpleBuilder.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6193783&bid=20207873)

**Review by vbradnitski — APPROVED** [review]

File: modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageRichBuilder.java [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+        html.text( "}" );
+
+        html.text( "pre.error span.marker {" );
+        html.text( " background: #A31012;" );
+        html.text( " color: #fff;" );
+        html.text( " text-shadow: 1px 1px 1px rgba(0, 0, 0, .3);" );
+        html.text( "}" );
+
+        html.close();
+        html.close();
```
Comment (anatol-sialitski) [inline modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageRichBuilder.java:201]: This line is duplicated

**Review by anatol-sialitski — APPROVED** [review]

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageSimpleBuilder.java  1
- modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageRichBuilder.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6193783&bid=20207873)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageSimpleBuilder.java  1
- modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/exception/ErrorPageRichBuilder.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6193783&bid=20207873)

**Pull request merged**